helm/fluent-logshipping-app/templates/configmap.yaml

```diff
@@ -82,7 +85,9 @@ data:
         Mem_Buf_Limit     ${input_memory_buffer_limit}
         Strip_Underscores on
         threaded          on
+  {{- end }}
 
+  {{- if or (has "audit" .Values.outputs.inputLogTypes) (has "sshd" .Values.outputs.inputLogTypes) (has "execve" .Values.outputs.inputLogTypes) }}
```

Comment:
So, the idea is to create this file only in cases where it will be included?
Then, the `{{end}}` should be after all the inputs are defined?